src/marqo/core/unstructured_vespa_index/constants.py

```diff
@@ -0,0 +1,7 @@
+# The threshold for a string to be considered short, long in unstructured Vespa.
+# Users will be able to filter on short strings, but not on long strings.
+SHORT_STRING_THRESHOLD = 20
+
+UNSUPPORTED_FIELD_NAME_LIST = ["::"]
```

Comment:
Name is a bit misleading. Unsupported field substring? Or you could just hard-code this value since it's the only prohibited subtring (unless it's used in multiple places, then must be a constant).